Source/FieldSolver/ImplicitSolvers/WarpXSolverVec.cpp

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2024 Justin Angus
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#include "FieldSolver/ImplicitSolvers/WarpXSolverVec.H"
+
+bool WarpXSolverVec::m_dot_mask_defined = false;
+amrex::Vector<std::array<std::unique_ptr<amrex::iMultiFab>,3>> WarpXSolverVec::m_dotMask;
```

Comment:
Why is the above line needed? Isn't `m_dotMask` already defined in `WarpXSolverVec.H`?

Reply:
`m_dotMask` is a static member. What's currently in the header is not a definition. The line here in a cpp file is a definition. This is what one has to do pre C++17. Nevertheless, since C++17, one can use `inline static`. So we could remove this line and the line above, and add `inline` to the lines in the header.

Reply:
Moved lines 9 and 10 to header file and added the inline to the declaration as suggested.